pkg/operator/ceph/cluster/controller.go

```diff
@@ -74,8 +74,8 @@ const (
 const (
 	// DefaultClusterName states the default name of the rook-cluster if not provided.
 	DefaultClusterName         = "rook-ceph"
-	clusterDeleteRetryInterval = 2 // seconds
-	clusterDeleteMaxRetries    = 15
+	clusterDeleteRetryInterval = 2 * time.Second
+	clusterDeleteMaxRetries    = 450
```

Comment:
If we're going to retry for this long, how about a retry interval of 5s with 180 retries?